test/support/fixtures/arguments_schema.ex

```diff
@@ -185,7 +182,17 @@ defmodule Absinthe.Fixtures.ArgumentsSchema do
           {:ok, "NESTED THAT #{thing}"}
 
         %{list_union: lists}, _ ->
-          {:ok, lists |> Enum.flat_map(&Map.values/1) |> Enum.join("&")}
+          # TODO FIX THIS
```

Comment:
this was just a quick way to test the list union, you could just update the test assertion

Reply:
yah I know. I liked what it was though.